test/arith2neura/add.mlir

```diff
@@ -1,8 +1,11 @@
-// RUN: mlir-neura-opt --lower-arith-to-neura %s | FileCheck %s
+// RUN: neura-compiler --neura-conversion %s | FileCheck %s --check-prefix=COMPILER
+// RUN: mlir-neura-opt --lower-arith-to-neura %s | FileCheck %s --check-prefix=OPT
```

Comment:
Excellent! Can we have the same thing for https://github.com/coredac/dataflow/blob/main/test/neura/ctrl/branch.mlir? Or any blocker for that?